### DISCUSSION TITLE
人员招收

### DISCUSSION THREAD

**Discussion by Doctor-Bright** [share121/inter-knot#2984] [其他]
## 我们正在进行人员招收

***

# 关于基金会

>人类到如今已经繁衍了25000年，只有最近的4000年是有意义的。

>所以，我们在将近25000年中在干嘛？我们躲在山洞中，围坐在小小的篝火边，畏惧那些我们不懂得的事物——那些关于太阳如何升起的解释，那些人头鸟身的怪物，那些有生命的石头。所以我们称他们为“神”和“恶魔”，并向他们祈求宽恕和祈祷拯救。

>之后，他们的数量在减少，我们的数量在增加。当我们恐惧的事物越来越少，我们开始更理智的看待这个世界。然而，不能解释的事物并没有消失，好像宇宙故意要表现出荒谬与不可思议一样。

>人类不能再生活在恐惧中。没有东西能保护我们，我们必须保护我们自己。当其他人在阳光下生活时，我们必须在阴影中和它们战斗，并防止它们暴露在大众眼中，这样其他人才能生活在一个理智的，普通的世界中。

**我们控制，我们收容，我们保护**
——[The Administrator](https://baike.baidu.com/item/The%20Administrator/24612359?fromModule=lemma_inlink)

![图片](https://img1.baidu.com/it/u=3246428639,1653988962&fm=253&fmt=auto&app=138&f=JPEG?w=577&h=500)

***

# 使命宣言：

>基金会于世界各地的暗处活动，目标是收容异常物品、个体及现象。其本身运作不受各个主要国家政府的司法管辖权、授权和委托的干扰。这些异常透过物理或心理危害对全球安全造成显著威胁

>基金会维持常态，从而使世界各地的平民得以生存并避免陷入恐惧、猜忌或失去信念的境况之中，并从地外、异次元和外层空间的威胁中维持人类的独立自主。

***

# **我们的任务有三方面**：

# 控制

>基金会控制异常以防止它们落入平民或敌对组织手中，透过广泛观察及监测并采取行动尽早拦截它们。

# 收容

>基金会收容异常以防止它们的影响或效应散播，通过迁移、掩饰、拆除它们或是通过抑制或阻止公众传播它们的知识的方式进行。

# 保护

>基金会保护人类免受异常的影响并保护异常本身直至它们被完全理解以及出现自它们的特性及行为上制定的新科学定理为止。如果异常被收容时被判定为过度危险，基金会或也会以将异常无效化或摧毁作为最后手段。

———————————————————————————————————————————————————————————

# 基金会行动

1. 特殊收容措施

>基金会维护着一个关于异常所需，通常称呼为“SCPs”的特殊收容措施的信息的庞大数据库。基金会主数据库装有异常的摘要以及在收容失效或其他事件中作维持或重新建立安全收容的紧急措施。
***
>异常有多种形式，或许是一件物品，一个实体，一个地点或一种独立的现象。这些异常被分类为一项特定的项目分级并收容于基金会众多设施中或在迁移被认为不可行的情况下就地收容。

2. 运营安保

>基金会以最高度的机密性运作。 所有基金会人员必须遵守安保权限等级以及按需知密与信息划分。被发现违反基金会安全协议的人员将被识别，拘留，并受到纪律处分。

3. 敌对机构及相关组织

>基金会并不是唯一一个拥有交互或使用异常的知识及能力的组织。尽管部分相关组织拥有相似的目的并可能与我们合作处理全球安全问题，但更多组织为投机主义者并以利益为依归，企图适应或利用异常以达到自己的目的。基金会人员被指示随时带着怀疑对待来自这些组织的个体，并在未经基金会领导层明确同意下与这些组织合作将会被处决或其他纪律处分。

***

# 如果您愿意奉献并在遵守上述条款的前提下，请您继续下滑

                                # [——[如不同意请立即停止浏览]——]
                                                         
                                  # 否则您将遭到处决
                                                   

                                 -DOWN OR EXIT-

     
![scplogo](https://github.com/user-attachments/assets/831dd5ed-acc1-4283-82e7-e44729122c3d)
***

![doc_O5(2)](https://github.com/user-attachments/assets/c31a6fe8-af41-49de-8f39-272ba49adeb8)

                                 【 你已经被警告过了】

***

![图片](https://scp-wiki.wdfiles.com/local--files/scp-001/fractal-mka.jpeg)

# 此消息已开启访问监控
### 模因抹杀触媒启动

                 ————**48小时内会有专人拨打您的手机或者向您的电子邮箱发送审核结果，必要时会提供上门服务。
                                          
                                          祝您生活愉快！**
                                                                   > ———— O5-7

                                                 Secure.Contain.Protect
                                                                      
![图片](https://bkimg.cdn.bcebos.com/pic/b21bb051f8198618367a107163bb39738bd4b21c48b5?x-bce-process=image/format,f_auto/resize,m_lfit,limit_1,h_672)

**Comment by lixiao189** [comment]
话说有一个叫作雷内的枫丹人是不是从你们这里拿到的资料发现了提瓦特世界式啊

**Comment by RaphaelHider** [comment]
零号空洞应该属于什么级别的收容物呢？Keter？<img src="https://bbs-static.miyoushe.com/static/2023/10/10/5bc71020b76055d271f9e744daaf880e_5037006894041683415.png" alt="拉尼_晕" width="75" >

**Comment by xiaosheng31** [comment]
别告诉我682突破收容来空洞了

**Comment by CyccyYan** [comment]
你这个人员招收……应该不是招收……D级吧……<img src="https://bbs-static.miyoushe.com/static/2023/11/16/9869e04ba6a0d0e6ed2418efc27d4b74_4411752698018709526.png" alt="苍角-诶？" width="75" >

**Comment by 1ctrl-cv** [comment]
照搬wiki是吧（

**Reply by Doctor-Bright** [reply]
确实
）

**Comment by CloudSwordSage** [comment]
要D级不，我兄弟说他要当D级<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by D111Z41SY** [comment]
我家楼下就有个SCP分部，对面楼还有不少分部。他们说自己Scan，Copy，Print，怎么不算SCP呢（）

**Comment by Timekeeper2006** [comment]
？？？
什么时候基金会也入驻新艾利都了！
我靠，我这个o5-4怎么不知道
那群老东西又不带我玩
难道就因为我离职了吗

**Comment by KUYUE404** [comment]
像我这种低等级绳匠，肯定是个D级炮灰

**Comment by WorldZHT** [comment]
招收的不会是D级吧
